l10n/en/navigation_refresh.ftl

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+navigation-refresh-menu = Menu
+navigation-refresh-mozilla = { -brand-name-mozilla }
+
+## About us
+
+navigation-refresh-about-us = About us
+navigation-refresh-close-about-us-menu = Close About us menu
+navigation-refresh-mozilla-manifesto = { -brand-name-mozilla } manifesto
+navigation-refresh-mozilla-foundation = { -brand-name-mozilla } Foundation
```

Comment:
(The fluent brand linter doesn't object, but may in the future, so to be on the safe side:)

```suggestion
navigation-refresh-mozilla-foundation = { -brand-name-mozilla-foundation }
```